pkg/setup/setup.go

```diff
@@ -31,8 +32,11 @@ import (
 	"github.com/spf13/afero"
 )
 
+const bundledCoreFilename="core.zip"
```

Comment:
Not sure if we need this as it's used from a single place and is not really meant to be exposed.